docs/contributing/developing.rst

```diff
@@ -282,6 +282,22 @@ This must be done after every change to the source files. To watch the source fi
 
     $ npm start
 
+Using the pattern library
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Wagtail’s UI component library is built with `Storybook <https://storybook.js.org/>`_ and `django-pattern-library <https://github.com/torchbox/django-pattern-library>`_. To run it locally,
+
+.. code-block:: console
+
+    $ export DJANGO_SETTINGS_MODULE=wagtail.tests.settings_ui
+    $ # Assumes the current environment contains a valid installation of Wagtail for local development.
+    $ ./wagtail/tests/manage.py migrate
+    $ ./wagtail/tests/manage.py createcachetable
+    $ ./wagtail/tests/manage.py runserver 0:8000
+    $ npm run storybook
```

Comment:
not sure if needed but it might be good to add a note that `npm run storybook` should be run in a different terminal/ssh or similar

Also, it would be good to have here that the default port for the actual viewing of the storybook UI is at `http://localhost:6006/`

Reply:
👍